media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -4934,3 +5073,106 @@ int mos_set_context_param_load_balance(struct mos_linux_context *ctx,
     free(balancer);
     return ret;
 }
+
+int mos_set_context_param_bond_master(struct mos_linux_context *ctx)
+{
+    int ret = 0;
+    struct {
+        struct i915_user_extension base;
+        __u16 engine_index;
+        __u16 num_siblings;
+        __u32 flags;
+        __u64 mbz64;
+        struct i915_engine_class_instance engines[1];
+    } __attribute__((packed)) balancer;
+
+
+    memclear(balancer);
+    balancer.base.next_extension = 0;
+    balancer.base.name = I915_CONTEXT_ENGINES_EXT_LOAD_BALANCE;
+    balancer.num_siblings = 1;
+    balancer.engines[0].engine_class = I915_ENGINE_CLASS_VIDEO;
+    balancer.engines[0].engine_instance = 0;
+
+    struct engines {
+        uint64_t extension;
+        struct i915_engine_class_instance class_instance[2];
+    } engines;
+    engines.extension = (uintptr_t)(&balancer);
+    engines.class_instance[0].engine_class = I915_ENGINE_CLASS_INVALID;
+    engines.class_instance[0].engine_instance = I915_ENGINE_CLASS_INVALID_NONE;
+    engines.class_instance[1].engine_class = I915_ENGINE_CLASS_VIDEO;
+    engines.class_instance[1].engine_instance = 0;
+
```

Comment:
Like in VENG, if you won't ever submit to index 1 in the map you don't need to configure it.

Reply:
yes, fixed.